azext_edge/tests/edge/orchestration/test_targets_unit.py

```diff
@@ -756,6 +745,40 @@ def test_custom_location_name_limits(target_scenario: dict, expected_error: Expe
             assert len(targets.custom_location_name) == len(target_scenario["custom_location_name"])
 
 
+def test_cross_rg_namespace_support():
```

Comment:
Instead of new tests, test cases in the existing tests such as test_init_targets can be used. The result being same coverage with less code. Additional assertions as necessary can be added to test_init_targets.